convexHull.cpp

```diff
@@ -0,0 +1,163 @@
+#include <iostream> 
+#include <stack> 
+#include <stdlib.h> 
+using namespace std; 
```

Comment:
* Add the brief description of Algo and its time complexity explicitly here. 
* Also, replace individual imports

```suggestion
#include <bits/stdc++.h>
using namespace std; 
```

Reply:
Updated the file according to your suggestions.